### PR TITLE
Add option to always show prayer flick helper

### LINKED ISSUE #962: Add a visual indicator to the tick metronome as well as a hotkey bind to turn it on/off
Not sure what would be the best type of visualizer but I was thinking either a flashing or pulsating orb.

### PR DESCRIPTION
**Pull request by ShaunDreclin**
Closes #962

Useful when doing tick manipulation activities other than prayer flicking

### CODE REVIEW

File: runelite-client/src/main/java/net/runelite/client/plugins/prayer/PrayerConfig.java [hunk 1]
```diff
@@ -44,28 +44,39 @@ default PrayerFlickLocation prayerFlickLocation()
 
 	@ConfigItem(
 		position = 1,
+		keyName = "prayerFlickAlwaysOn",
+		name = "Pray flick helper always on",
```
Comment (deathbeam) [inline runelite-client/src/main/java/net/runelite/client/plugins/prayer/PrayerConfig.java]: Change this to "Never hide prayer flick helper"

Comment (ShaunDreclin) [reply runelite-client/src/main/java/net/runelite/client/plugins/prayer/PrayerConfig.java]: done

**Comment by Nightfirecat** [comment]
This closes #962

**Comment by ShaunDreclin** [comment]
> lol

... why are you commenting on a 7 year old pull request?

**Comment by rubydagrim** [comment]
Please man, please I get paid once a month please if you'll kind of show me how to use this website if you're gonna show me how to use GitHub dude I'll pay you a little bit at least

**Pull request merged**